easybuild/easyconfigs/m/Myokit/Myokit-1.32.0-fosscuda-2020b.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'PythonPackage'
+
+name = 'Myokit'
+version = '1.32.0'
+
+homepage = 'http://myokit.org'
+description = """Myokit is an open-source Python-based toolkit that facilitates modeling and simulation of cardiac
+cellular electrophysiology."""
+
+toolchain = {'name': 'fosscuda', 'version': '2020b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
```

Comment:
```suggestion
```

Reply:
No longer needed since we've added https://github.com/easybuilders/easybuild-easyblocks/pull/2364